### PR TITLE
add/not add robots.txt prompt and new deployment

### PR DESCRIPTION
**Pull request by ricardovanlaarhoven**

### CODE REVIEW

File: generator/templates/Deployment/_github/workflows/deploy.test.yml [hunk 1]
```diff
@@ -53,11 +54,48 @@ jobs:
...
+              cd ~/domains/example.com/public_html
+              rm -rf ./old
+              mkdir -p ./new
+              mkdir -p ./old
       - name: deployment
         run: |
           cd dist
-          rsync -ave "ssh -o ProxyJump=jumphost@jumphost.kingscode.nl" ./ user@host:~/domains/example.com/public_html
+          rsync -ave "ssh -o ProxyJump=jumphost@jumphost.kingscode.nl" ./ user@host:~/domains/example.com/public_html/new
+      - name: move previous deployment to ./old and move ./new deployment to ./
```
Comment (Deluze) [inline generator/templates/Deployment/_github/workflows/deploy.test.yml:88]: Why is this split up in 2 actions?

You don't need to create a `new` directory for the sake of directly copying everything from it in the end and then removing the directory. The html root _is_ the new directory.

-> create old dir
-> move to old dir
-> clear target dirs
-> copy dist to html root dir

Comment (koenhoeijmakers) [reply generator/templates/Deployment/_github/workflows/deploy.test.yml:88]: While moving to old dir there will be no content available for a short period of time, we want no downtime deployments.

**Review by koenhoeijmakers — APPROVED** [review]

File: generator/templates/Deployment/_github/workflows/deploy.test.yml (hunk 1 above)
Comment (ricardovanlaarhoven) [reply generator/templates/Deployment/_github/workflows/deploy.test.yml:88]: And it prevents failures, when we remove all the old files first and the build fails, then is nothing lost.

Comment (Deluze) [reply generator/templates/Deployment/_github/workflows/deploy.test.yml:88]: Not like that would happen, because the action won't run if it fails

**Review by Deluze — APPROVED** [review]

**Pull request merged**